CMakeLists.txt

```diff
@@ -116,6 +116,9 @@ include_directories(${CMAKE_CURRENT_SOURCE_DIR}/profiling/all)
 set(COMMON_HEADERS_PATH ${CMAKE_CURRENT_BINARY_DIR}/common)
 include_directories(${COMMON_HEADERS_PATH})
 
+# Allow all tools to include any file.
+include_directories(${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
In isolation this looks reasonable but it clashes with the other include directories (I find it weird that we have to specify `common/` for some and can omit it for the configured files.)

Reply:
What do you propose then?

Reply:
@dalg24 ?

Reply:
I suppose this is more "contained"
```suggestion
# Allow all tools to include any file.
include_directories(${CMAKE_CURRENT_SOURCE_DIR}/common)
```

Reply:
OK, done.